repository/repo-operations.go

```diff
@@ -265,6 +277,13 @@ func updateRepo(config *config.GitXargsConfig, repositoryDir string, worktree *g
 		return pushBranchErr
 	}
 
+	// If the user passed --skip-pull-requests, don't open a pull request
+	if config.SkipPullRequests {
+		wg.Done()
+		p.Increment()
```

Comment:
This needs to be updated.

Reply:
Thank you! Fixed in 105793bafa223e9ca26bf39d3ed8d6ffd526ce2c

Reply:
Oh I think you needed this if condition to fully skip opening the PR? Otherwise, it will flow through to open the PR, right?